app.js

```diff
@@ -237,24 +326,22 @@ async function watchRandomStreamers(browser, page) {
     }
 
     // Is this streamer still streaming?
-    const channelStatus = await page
-      .$eval(CHANNEL_STATUS_QUERY, (el) => el.textContent)
-      .catch(() => "");
+    const channelStatus = (await query(page, CHANNEL_STATUS_QUERY)).text().trim().toUpperCase();
     info("Channel status: " + channelStatus);
     // We use `startsWith` because sometimes we get LIVELIVE
     // Also `toUpperCase` because sometimes we get LiveLIVE
     // This is because there are two elements with that class name
     // One below the player and one "in" the player
-    if (!channelStatus.toUpperCase().startsWith("LIVE")) {
+    if (!channelStatus.includes("LIVE")) {
       error("Streamer is offline");
       await page.waitFor(jitter(1000));
       continue;
     }
 
     // Does the streamer have drops enabled?
-    const dropsEnabled = await page
-      .$eval('[data-a-target="Drops Enabled"]', () => true)
-      .catch(() => false);
+    //Updated by AlexSimpler
+    const dropsEnabled = const dropsEnabled = (await query(page, DROP_STATUS)).text();
```

Comment:
I think this line is a typo?

Reply:
yes, my bad, some rather bad copying from me.